domenetjenester/brevbestiller/src/test/java/no/nav/foreldrepenger/melding/brevmapper/brev/innvilgelsefp/BeregningsgrunnlagMapperTest.java

```diff
@@ -158,24 +197,38 @@ public void skal_finne_at_brutto_beregningsgrunnlag_ikke_er_brukt() {
         assertThat(resultat).isFalse();
     }
 
-    private BeregningsgrunnlagPeriode lagBeregningsgrunnlagPeriode() {
+    private BeregningsgrunnlagPeriode lagBeregningsgrunnlagPeriode(List<BeregningsgrunnlagPrStatusOgAndel> andelsliste) {
         return BeregningsgrunnlagPeriode.ny()
                 .medDagsats(STANDARD_PERIODE_DAGSATS)
                 .medBruttoPrÅr(FRILANSER_BRUTTO_PR_ÅR)
                 .medAvkortetPrÅr(AVKORTET_PR_ÅR)
-                .medBeregningsgrunnlagPrStatusOgAndelList(lagBgpsaListe())
+                .medBeregningsgrunnlagPrStatusOgAndelList(andelsliste)
                 .build();
     }
 
     private List<BeregningsgrunnlagPrStatusOgAndel> lagBgpsaListe() {
-        return of(lagBgpsaBruttoFrilanser(), lagBgpsaAvkortetArbeidstaker());
+        return of(lagBgpsandel(FRILANSER_BRUTTO_PR_ÅR, FRILANSER_DAGSATS, AktivitetStatus.FRILANSER, false),
+                lagBgpsaAvkortetArbeidstaker());
     }
 
-    private BeregningsgrunnlagPrStatusOgAndel lagBgpsaBruttoFrilanser() {
+    private List<BeregningsgrunnlagPrStatusOgAndel> lagBgspaListe2() {
```

Comment:
Er det mulig å navngi med noe som sier mer om hva som er spesielt med lista, i stedet for bare tall? (gjerne da på alle tre)

Reply:
enig, kan se hva jeg finner på av bedre navngivning :-)